packages/fluentui/react-northstar/src/index.ts

```diff
@@ -200,6 +200,8 @@ export * from './components/Card/CardFooter';
 export * from './components/Card/CardColumn';
 export * from './components/Card/CardExpandableBox';
 
+export * from './components/Datepicker/Unstable_Datepicker';
```

Comment:
```suggestion
export * from './components/Datepicker/Unstable_Datepicker';
export * from './components/Datepicker/DatepickerCalendar';
```

`DatepickerCalendar` should be also exported to pass tests